### DISCUSSION TITLE
More styling options for "Post Info" widget

### DISCUSSION THREAD

**Discussion by MalikKhadje** [elementor/elementor#24611] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Dear Elementor Pro Team,

I am writing to suggest an enhancement for the "Post Info" widget within Elementor Pro that I believe would significantly benefit users who seek more control and creativity in their web design.

Current Limitations:
Right now, the "Post Info" widget offers a limited range of styling options. Users can adjust basic features such as color, alignment and spacing. However, the options for making it unique and fitting for more complex designs are somewhat limited. Specifically, there is a lack of options to customize styles for the dynamic content itself.

The Issue:
Not having many style choices makes it hard to create a unified look that matches the website's style. This limitation makes it difficult to organize the visual design and keep everything on the website looking the same.

The Benefit:
Adding these features would help overcome the current design limitations. It would give users the freedom to make their post information sections more interesting and eye-catching.

### Describe the solution you'd like.

Desired enhancements for the 'Post Info' widget content (not the widget container itself) include:

Border Styles: Customizable borders for individual metadata elements like author, date, and comments, with options for style, width, and color.
Padding: Independent padding adjustments for each element to refine spacing.
Hover Effects: Element-specific hover states with options for color change and text-decoration.
These changes would offer greater design flexibility for the 'Post Info' content without affecting the overall widget's container.

### Describe alternatives you've considered.

I've experimented with different ways to show post information, hoping to have more styling choices. However, based on my research, the "Post Info" widget stands out as the most convenient and efficient way to display this information. In fact, it might be the only option that makes displaying post information both easy and fast.

All we need is a small enhancement that can lead to significant improvements.

### Additional context

![afbeelding_2023-12-19_203211542](https://github.com/elementor/elementor/assets/109419802/7f5d239c-270f-4827-b650-d5e7a2e325f0)

Here is an image, of the current style tab for post info content.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by catchlightWeb** [comment]
With the Loop Builder and dynamic data on practically everything, this is kinda negated now, I believe. If you want a Custom Post Info block just roll it yourself.
